### PR TITLE
Comments text tweaks

### PR DESCRIPTION
**Pull request by wedebe**

### CODE REVIEW

**Comment by jbleyel** [comment]
Please create only one pull request with all the text tweaks .. Thanks

**Comment by jbleyel** [comment]
This commit also changes the code .. please don't mix comment changes with code changes

**Comment by jbleyel** [comment]
You may have used a old version of the file.

File: lib/dvb/epgcache.cpp [hunk 1]
```diff
@@ -1607,8 +1609,9 @@ int handleEvent(eServiceEvent *ptr, ePyObject dest_list, const char* argstring,
 				Py_DECREF(nowTime);
 			Py_DECREF(convertFuncArgs);
 			Py_DECREF(dest_list);
-			PyErr_SetString(PyExc_Exception, "[eEPGCache] handleEvent: error in convertFunc execute");
-			//eDebug("[eEPGCache] handleEvent: error in convertFunc execute");
+			PyErr_SetString(PyExc_StandardError,
```
Comment (jbleyel) [inline lib/dvb/epgcache.cpp:1612]: PyExc_StandardError is wrong!
And you don't need eDebug if you have a python exception.

File: lib/dvb/epgcache.cpp [hunk 2]
```diff
@@ -2442,7 +2449,15 @@ PyObject *eEPGCache::search(ePyObject arg)
 			ePyObject obj = PyTuple_GET_ITEM(arg,0);
 			if (PyString_Check(obj))
 			{
+#if PY_VERSION_HEX < 0x02060000
```
Comment (jbleyel) [inline lib/dvb/epgcache.cpp:2452]: ? what's this

File: lib/dvb/epgcache.cpp [hunk 3]
```diff
@@ -3197,14 +3232,14 @@ void eEPGCache::crossepgImportEPGv21(std::string dbroot)
...
-		eDebug("[eEPGCache] cannot open crossepg aliases db");
+	eDebug("[eEPGCache] cannot open crossepg aliases db");
 		fclose(headers);
 		fclose(descriptors);
 		return;
 	}
 
 	/* read headers */
-	ret = fread(tmp, 13, 1, headers);
+	fread (tmp, 13, 1, headers);
```
Comment (jbleyel) [inline lib/dvb/epgcache.cpp:3242]: don't remove ret =

**Comment by wedebe** [comment]
> You may have used a old version of the file.

Agh, my apologies, I pasted into GitHub's web interface. Code changes were not intended, just comments.

**Pull request closed without merging**